Add <NativeLibrary /> documentation (#980)
dotnet/runtimelab: docs/using-nativeaot/interop.md

</ItemGroup>
```

### Linking

> To statically link against a native library, you'll need to specify `<NativeLibrary>filename</NativeLibrary>` pointing to `.lib` file

Reviewer: ```suggestion
To statically link against a unmanaged library, you'll need to specify `<NativeLibrary>filename</NativeLibrary>` pointing to `.lib` file on Windows and `.a` file on Unix
```

Examples:

```xml
<ItemGroup>
  <!-- Generate direct PInvokes for Dependency -->
  <DirectPInvoke Include="Dependency" />

>   <!-- Specify .lib file -->
>   <NativeLibrary Include="Dependency.lib" />

Reviewer: ```suggestion
  <!-- Specify library to link against -->
  <NativeLibrary Include="Dependency.lib" Condition="$(RuntimeIdentifier.StartsWith('win'))" />
  <NativeLibrary Include="Dependency.a" Condition="!$(RuntimeIdentifier.StartsWith('win'))" />
```

</ItemGroup>
```

### Linking

> To statically link against a unmanaged library, you'll need to specify `<NativeLibrary>filename</NativeLibrary>` pointing to `.lib` file on Windows and `.a` file on Unix

Reviewer 2: ```suggestion
To statically link against a unmanaged library, you'll need to specify `<NativeLibrary Include="filename" />` pointing to `.lib` file on Windows and `.a` file on Unix
```
